openlibrary/components/LibraryExplorer/components/LibraryToolbar.vue

```diff
@@ -462,12 +462,6 @@ export default {
   }
 
   .chunky-icon {
-    padding: 4px;
-    margin: 4px;
-    width: 52px;
-    transition: background-color .2s;
-    text-align: center;
```

Comment:
Let's restore most of these; they should work once the `.chunky-icon` class is added back.

(The rough idea, is any CSS that would apply to a `chunky-icon` should be here. CSS that applies to specifically a button in a tab-bar context should be in `.tab-bar > button`)